sda-commons-web-autoconfigure/src/main/resources/org/sdase/commons/spring/boot/web/defaults.properties

```diff
@@ -6,3 +6,9 @@ server.error.whitelabel.enabled=false
 server.tomcat.basedir=${java.io.tmpdir}/tomcat
 
 request.body.max.size=1 MB
+
+management.tracing.sampling.probability=1.0
```

Comment:
Does this mean, that we propagate every trace? I'm not sure if that could cause too much tracing traffic. Do you know the previous default and the default in Dropwizard Commons?

Reply:
dw commons uses 1.0 as default from class `io.opentelemetry.sdk.autoconfigure.TracerProviderConfiguration` property `DEFAULT_TRACEIDRATIO_SAMPLE_RATIO = 1.0d;`